xsd/netex_framework/netex_reusableComponents/netex_equipmentVehiclePassenger_version.xsd

```diff
@@ -16,14 +16,13 @@
 				<Date>
 					<Modified>2011-12-16</Modified>
 				</Date>
-
 				<Description>
 					<p>NeTEx - Network Exchange. This subschema defines passenger vehicle EQUIPMENT </p>
 				</Description>
 				<Format>
 					<MediaType>text/xml</MediaType>
 					<Syntax>http://www.w3.org/2				<Date>
-					<Modified>2019-03-25</Modified>Fix #41 by Skinkie from 2019.01.07. Fix typo on MobilityList. 
+							<Modified>2019-03-25</Modified>Fix #41 by Skinkie from 2019.01.07. Fix typo on MobilityList. 
```

Comment:
@nick-knowles could you change the text alignment?

Reply:
```suggestion
					<Modified>2019-03-25</Modified>Fix #41 by Skinkie from 2019.01.07. Fix typo on MobilityList. 
```